apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -153,7 +153,7 @@ def get_query_server_config(name='beeswax', connector=None):
         if HIVE_DISCOVERY_LLAP.get():
           LOG.debug("Checking zookeeper for discovering Hive LLAP server endpoint")
           zk = KazooClient(hosts=libzookeeper_conf.ENSEMBLE.get(), read_only=True)
-          zk.start()
+          zk.start(ZOOKEEPER_CONN_TIMEOUT.get())
```

Comment:
@quadoss timeout is a keyword arguments: https://github.com/cloudera/hue/blob/master/desktop/core/ext-py/kazoo-2.8.0/kazoo/client.py#L619.

```
zk.start(timeout=ZOOKEEPER_CONN_TIMEOUT.get())
```

Reply:
Thanks @wing2fly for the review and highlighting the change needed.  While it works without 'timeout=', I had made the needed changes to comply with whats there in the API doc.